src/test/java/com/twilio/jwt/accesstoken/AccessTokenTest.java

```diff
@@ -336,4 +338,18 @@ public void testVoiceTokenWithoutIncoming() {
         Assert.assertEquals("AP123", outgoing.get("application_sid"));
         Assert.assertEquals("bar", outgoingParams.get("foo"));
     }
+
+    @Test()
+    public void testNullValues() throws JsonProcessingException {
```

Comment:
Changes looks fine, but there's a lot of duplicated code in this test class. Could you refactor the obvious bits? Also, the other tests use a JWT parser to validate the payload rather than an object mapper. I prefer making this one uniform with the others so we know the payload conforms rather than what an object mapper produces.